### PR TITLE
Hello

### PR DESCRIPTION
**Pull request by franclobo**
**Set up Hello Microverse**

1. Create a branch called "hello".
2. Create a folder .github/workflows.
3. Copy the document linters.yml.
4. Copy the document .hintrc.
5. Copy the document .stylelintrc.jason.
6. Add a RESUME.md file.
7. Add a .gitignore file.

**8. Add 3 files (each one in a separate commit):**

- A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.#
- A .css file that includes one class with styles for your header (make it in your favorite color).#
- A test.md file with the text "This file should be ignored by git".#

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```
Comment (gemmen29) [inline test.md]: Kindly include this file ` test.md ` in your `.gitignore` file. This file should not exist in the repo and should be ignored by git as it's in the requirement for this project.

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```
Comment (gemmen29) [inline README.md]: Kindly use the Microverse README template for the [README file](https://github.com/microverseinc/readme-template) with its styling and add the same information you added to that template.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Hello</title>
+</head>
+<body>
+    <h1 id = title>"Hello Microverse"</h1>
```
Comment (gemmen29) [inline index.html]: [OPTIONAL] kindly wrap id with " ", so it will be like:
```
 <h1 id ="title">Hello Microverse</h1>
```
For valid HTML syntax.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @franclobo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (franclobo) [reply README.md]: May I have to change the README.md file in the main branch?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Hello</title>
+</head>
+<body>
+    <h1 id ="title">Hello Microverse</h1>
+</body>
```
Comment (OLIPLICHE) [inline index.html:12]: Great 👏!! You implemented the change💯

File: README.md [hunk 5]
```diff
@@ -1 +1,9 @@
...
+This is my firt project like a Microverse student.
+# Built with
+HTML & CSS
+# Author
+Francisco Borja
+GitHub: @franclobo
+Twitter: @pancho2788
```
Comment (OLIPLICHE) [inline README.md]: - Please open this [Link](https://github.com/microverseinc/readme-template)
- What you find in the Link is the ReadMe Template which needs to be used.
-  Remember to customize this ReaMe Template to your project, the README file should be descriptive 👍

File: test.md (hunk 1 above)
Comment (OLIPLICHE) [inline test.md]: - Please test.md folder should be in the .gitignore file and not in the Github repository. 👍

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @franclobo, 

image


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Great!!!! for implementing the changes 👏 👏👏

 ## Required Changes ♻️

- [ ] Check the comments under the review⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (gemmen29) [reply README.md]: No, do all changes in the branch and you can merge it to main after approval.

**Review by Wes-Isaac — APPROVED** [review]
Hi @franclobo,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**